packages/bundler/src/modules/BundleManager.ts

```diff
@@ -441,10 +441,20 @@ export class BundleManager implements IBundleManager {
       }
       mergeStorageMap(storageMap, validationResult.storageMap)
 
-      const mergeOk = this.mergeEip7702Authorizations(entry, sharedAuthorizationList)
-      if (!mergeOk) {
-        debug('unable to add bundle as it relies on an EIP-7702 tuple that conflicts with other UserOperations')
-        continue
+      const authorizationList = getAuthorizationList(entry.userOp)
```

Comment:
Why did you inline this method? And any way, the `mergeEip7702Authorizations` is still there. I don't see a point in this change.

Reply:
getAuthorizationList is for simpler support for erc4337 and rip7562: it always returns a list, and empty one of no tuple exists. This saves checking on each use.